### PR TITLE
licensed: update test

### PR DESCRIPTION
**Pull request by chenrui333**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `HOMEBREW_NO_INSTALL_FROM_API=1 brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

- relates to https://github.com/Homebrew/homebrew-core/pull/173308

### CODE REVIEW

File: Formula/l/licensed.rb [hunk 1]
```diff
@@ -55,13 +55,18 @@ def install
...
+        bundler:
+          # in CI, bundler is set up pretty wonky and isn't found at the expected location
+          - "bundler"
+
+      cache_path: .licenses_test
     EOS
 
     assert_match "Caching dependency records for test",
-                        shell_output(bin/"licensed cache")
+                        shell_output(bin/"licensed cache", 1)
```
Comment (Bo98) [inline Formula/l/licensed.rb:70]: Expecting it to fail seems odd

Comment (chenrui333) [reply Formula/l/licensed.rb:70]: yes, I am gonna file an issue about this.

Comment (chenrui333) [reply Formula/l/licensed.rb:70]: looks like the licensed installed from `gem install licensed` output `0` for me

```
$ gem install licensed
...
$ licensed --version
4.4.0

$ licensed cache
Caching dependency records for test

$ echo $?
0
```

**Comment by chenrui333** [comment]
I can no longer reproduce in my local

```
$ /opt/homebrew/Cellar/licensed/4.4.0_3/bin/licensed cache
Caching dependency records for test

$ echo $?
0
```

**Comment by chenrui333** [comment]
looks like it was just flakiness, close for now.

**Pull request closed without merging**